src/main/java/eu/bbmri_eric/negotiator/api/controller/v3/AccessFormController.java

```diff
@@ -1,37 +1,76 @@
 package eu.bbmri_eric.negotiator.api.controller.v3;
 
 import eu.bbmri_eric.negotiator.dto.access_form.AccessFormDTO;
+import eu.bbmri_eric.negotiator.dto.access_form.ElementCreateDTO;
+import eu.bbmri_eric.negotiator.dto.access_form.ElementMetaDTO;
+import eu.bbmri_eric.negotiator.dto.access_form.SectionCreateDTO;
+import eu.bbmri_eric.negotiator.dto.access_form.SectionMetaDTO;
+import eu.bbmri_eric.negotiator.mappers.AccessFormElementAssembler;
 import eu.bbmri_eric.negotiator.mappers.AccessFormModelAssembler;
+import eu.bbmri_eric.negotiator.mappers.AccessFormSectionAssembler;
 import eu.bbmri_eric.negotiator.service.AccessCriteriaSetService;
+import eu.bbmri_eric.negotiator.service.AccessFormElementService;
 import eu.bbmri_eric.negotiator.service.AccessFormService;
+import eu.bbmri_eric.negotiator.service.AccessFormsSectionService;
 import io.swagger.v3.oas.annotations.Operation;
 import io.swagger.v3.oas.annotations.tags.Tag;
-import org.springframework.beans.factory.annotation.Autowired;
+import jakarta.validation.Valid;
+import org.modelmapper.ModelMapper;
 import org.springframework.data.domain.Page;
 import org.springframework.data.domain.PageRequest;
+import org.springframework.hateoas.CollectionModel;
 import org.springframework.hateoas.EntityModel;
+import org.springframework.hateoas.MediaTypes;
 import org.springframework.hateoas.PagedModel;
 import org.springframework.http.HttpStatus;
 import org.springframework.http.MediaType;
-import org.springframework.web.bind.annotation.CrossOrigin;
 import org.springframework.web.bind.annotation.GetMapping;
 import org.springframework.web.bind.annotation.PathVariable;
+import org.springframework.web.bind.annotation.PostMapping;
+import org.springframework.web.bind.annotation.PutMapping;
+import org.springframework.web.bind.annotation.RequestBody;
 import org.springframework.web.bind.annotation.RequestMapping;
 import org.springframework.web.bind.annotation.RequestParam;
 import org.springframework.web.bind.annotation.ResponseStatus;
 import org.springframework.web.bind.annotation.RestController;
 
 @RestController
-@RequestMapping("/v3")
-@CrossOrigin
+@RequestMapping(
+    value = "/v3",
+    produces = MediaTypes.HAL_JSON_VALUE,
+    consumes = MediaType.APPLICATION_JSON_VALUE)
```

Comment:
This is causing an exception when getting the negotiations. 
Using postman to GET /api/v3/access-forms it results in 
```
{
    timestamp": "2024-04-23",
    "status": 415,
    "error": "Unsupported Media Type",
    "trace": "org.springframework.web.HttpMediaTypeNotSupportedException: Content-Type ....
}
```
I guess it expects json body that the GET call doesn't need to send

Reply:
yes, fixed in the next PR but I can remove it here as well